src/main/webapp/style.css

```diff
@@ -165,6 +165,18 @@ h1 {
   object-fit: cover;
 }
 
+#info-container * {
+  overflow: auto;
```

Comment:
When will it overflow?

Reply:
My initial thought was that it would overflow once I add the icon beside it, it doesn't. I will be removing it.